### PR TITLE
[CCTRI-4048] Update Makefile and format files

### PR DESCRIPTION
**Pull request by grzegorzbielecki**

### CODE REVIEW

**Review by Tempos — APPROVED** [review]

File: code/Pipfile [hunk 1]
```diff
@@ -5,16 +5,16 @@ verify_ssl = true
...
 cryptography = "==41.0.3"
-Flask = "==2.2.5"
-marshmallow = "==3.19.0"
-PyJWT = "==2.6.0"
+Flask = "==2.3.3"
+marshmallow = "==3.20.1"
+PyJWT = "==2.8.0"
 requests = "==2.31.0"
 
 [dev-packages]
```
Comment (jordan5rov) [inline code/Pipfile:13]: we need to think about adding the 'pip-audit' library here for the dependencies investigations

**Review by jordan5rov — CHANGES_REQUESTED** [review]

File: code/Pipfile (hunk 1 above)
Comment (grzegorzbielecki) [reply code/Pipfile:13]: I will check it on a new branch

File: code/Pipfile [hunk 2]
```diff
@@ -5,16 +5,16 @@ verify_ssl = true
...
-flake8 = "==6.0.0"
-pytest = "==7.2.2"
+black = "==23.9.1"
+coverage = "==7.3.1"
+flake8 = "==6.1.0"
+pytest = "==7.4.2"
 
 [requires]
-python_version = "3.11"
+python_version = "3.11"
```
[the remark this answers is not in the document]
Comment (grzegorzbielecki) [reply code/Pipfile]: Done

File: code/Pipfile (hunk 1 above)
Comment (grzegorzbielecki) [reply code/Pipfile:13]: Done

**Review by jordan5rov — APPROVED** [review]

**Pull request merged**